### DISCUSSION TITLE
可以加入数字标注等功能

### DISCUSSION THREAD

**Discussion by surgue** [Ljzd-PRO/buttplug-dg-lab#1] [Ideas]
感谢作者自编译的应用程序，在用的时候有一些不方便的地方，比如控制条没有数字标注，看不见当前值。可以尝试加入设置波形的功能或者作者可以给几个默认波形选项，每次连接上都会重置有些麻烦。最后一点是拖动控制条的时候会被迫自己试强度，可以添加一个开关键或者类似的功能。

**Comment by surgue** [comment]
能否使用central里的socket功能连接dglab呢

**Reply by Ljzd-PRO** [reply]
> 能否使用central里的socket功能连接dglab呢

不能，或者说得自己在central这一段再增加一些逻辑，同时得搭建个dg-lab的socket服务端，因为它socket协议比较复杂
